src/util/ssa_expr.h

```diff
@@ -16,6 +16,8 @@ Author: Daniel Kroening, kroening@kroening.com
 class ssa_exprt:public symbol_exprt
 {
 public:
+  ssa_exprt() : symbol_exprt("", typet()) {}
```

Comment:
This is pretty much a show stopper: why is this needed? Presumably a side effect of how objects are created in a sharing map, but can the sharing map please be fixed instead?

Reply:
Maybe somewhere between @danpoe and @smowton: can you look into this and see whether we can get away without it?

Reply:
Sure, I'll take a look

Reply:
I was meaning to fix this once we have decided whether to merge this PR. Currently the sharing map creates a dummy object (using the default constructor of the stored value type) when for a `delta_view_itemt` the key is only in one of the maps. We'll can probably use use an `optionalt` instead.

Reply:
https://github.com/diffblue/cbmc/pull/4612 @danpoe

Reply:
#4612 is merged.